playground/src/app.critical.scss

```diff
@@ -207,6 +256,39 @@
       }
     }
   }
+
+  &--light {
+    .panel-header {
+
+      * {
+        color: #000;
+      }
+
+      @media (min-width: 786px) {
+        & > * {
+          border-right: 1px solid rgba(0, 0, 0, 0.15);
+
+          &:last-child {
+            margin-left: auto;
+            border-left: 1px solid rgba(0, 0, 0, 0.15);
+            border-right: 0;
+          }
+        }
+      }
+
+      select,
+      .button {
+        &:hover,
+        &:active {
+          background-color: rgba(0, 0, 0, 0.02);
+        }
+      }
+
+      svg {
+        fill: #20212a;
```

Comment:
In terms of consistency: our next fitting color is `color('charade')` or #20202A. They are almost the same with `rgb(32, 32, 42)` and `rgb(32, 33, 42)`